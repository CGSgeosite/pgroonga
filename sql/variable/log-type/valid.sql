-- To load PGroonga
SELECT pgroonga.command('status')::json->0->0;

SHOW pgroonga.log_type;
SET pgroonga.log_type = 'windows_event_log';
SHOW pgroonga.log_type;
SET pgroonga.log_type = default;
SHOW pgroonga.log_type;
