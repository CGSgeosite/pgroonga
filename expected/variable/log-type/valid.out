-- To load PGroonga
SELECT pgroonga.command('status')::json->0->0;
 ?column? 
----------
 0
(1 row)

SHOW pgroonga.log_type;
 pgroonga.log_type 
-------------------
 file
(1 row)

SET pgroonga.log_type = 'windows_event_log';
SHOW pgroonga.log_type;
 pgroonga.log_type 
-------------------
 windows_event_log
(1 row)

SET pgroonga.log_type = default;
SHOW pgroonga.log_type;
 pgroonga.log_type 
-------------------
 file
(1 row)

